examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino

```diff
@@ -20,8 +20,8 @@
 #include <Firebase.h>
 
 // create firebase client.
-Firebase fbase("example.firebaseio.com")
-         .auth("secret_or_token");
+Firebase fbase = Firebase("example.firebaseio.com")
```

Comment:
It is a little more combersome but I can see how it is useful with the setup() loop() cycle. We could provide both options.

How about we call it "setup()" instead of begin though. to match the arduino function name.